src/gui/qgsabstractrelationeditorwidget.h

```diff
@@ -183,6 +183,18 @@ class GUI_EXPORT QgsAbstractRelationEditorWidget : public QWidget
      */
     bool multiEditModeActive() const;
 
+  signals:
+
+    /**
+     * Emit this signal, whenever the related features changed.
+     * This happens for example when related features are added, removed,
+     * linked or unlinked.
+     *
+     * \param vectorLayer The modified layer
```

Comment:
I don't see this param. Is it missing here?

Reply:
Oh I removed and did not updated the doc... I'll update it.